ctapipe/containers.py

```diff
@@ -111,8 +111,10 @@ class HillasParametersContainer(Container):
     r = Field(nan * u.m, "radial coordinate of centroid", unit=u.m)
     phi = Field(nan * u.deg, "polar coordinate of centroid", unit=u.deg)
 
+    length = Field(nan * u.m, "standard deviation along the major-axis", unit=u.m)
```

Comment:
This just duplicates the field. THere is a suffix or prefix missing. `length_uncertainty`?